tests/test_gisaid_utils.py

```diff
@@ -1,83 +1,143 @@
 import unittest
-from covizu.utils.gisaid_utils import *
+from covizu.utils.gisaid_utils import load_gisaid, batch_fasta, extract_features, sort_by_lineage
 
 
-def callback(s):
-    print(s)
+def callback(stuff):
```

Comment:
Stuff is probably not the right variable name

Reply:
For callback functions I usually use `msg`